### ISSUE TITLE
Generate xUnit reports

### ISSUE THREAD

**Issue by Thiht** [fossas/fossa-cli#110]
Would you consider supporting the generation of [xUnit](https://xunit.github.io/) reports?

They easily integrate with Jenkins, SonarQube, and more!

**Comment by ilikebits** [comment]
Yes, this sounds like a great idea! We'll put this on our roadmap, happy to accept PRs.
